README.md

```diff
@@ -15,4 +16,21 @@ npm init @eslint/config
 ```bash
 # use npx
 npx @eslint/create-config
-```
\ No newline at end of file
+```
+
+If you want to use the specified shared configs:
+
+```bash
+# use `eslint-config-eslint` shared config
+npm init @eslint/config --config eslint
+# or (`eslint-config` prefix is optional)
+npm init @eslint/config --config eslint-config-eslint
+```
+
+The `--config` flag also supports passing in arrays:
+
+```bash
+npm init @eslint/config --config eslint,standard
+# or
+npm init @eslint/config --config eslint --config standard
```

Comment:
This doesn't seem to work. I'm getting `TypeError: argv.config.split is not a function` because in this case `argv.config` is an array.